src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -450,28 +452,34 @@ - (void)invokeCommandWithEndpointID:(NSNumber *)endpointID
                           clusterID:(NSNumber *)clusterID
                           commandID:(NSNumber *)commandID
                       commandFields:(id)commandFields
-                     expectedValues:(NSArray<NSDictionary<NSString *, id> *> *)expectedValues
+                     expectedValues:(NSArray<NSDictionary<NSString *, id> *> * _Nullable)expectedValues
               expectedValueInterval:(NSNumber *)expectedValueInterval
```

Comment:
`expectedValueInterval` is also nullable, right?

What should happen if one of `expectedValues` and `expectedValueInterval` is nil and the other is not?  I'd assume they should either both be nil or both not be nil, and we should error out in the other cases...  But either way we should document the behavior in the header.

Reply:
Yes indeed. I was copying over the _Nullable from the header for expectedValues and missed this. I'll add it.